src/utils/google-signin-utils.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Sentinel used to tell parent that we are communicating with it. */
+export const SENTINEL = 'google_signin';
+/** Command name for when the parent frame is ready for iframe start. */
+export const PARENT_READY_COMMAND = 'parent_frame_ready';
+/** Command name for when the Google Sign-in iframe is ready. */
+export const INTERMEDIATE_IFRAME_READY_COMMAND = 'intermediate_iframe_ready';
+/** Command name for when the user's entitlements are returned from the publisher. */
+export const ENTITLEMENTS_READY_COMMAND = 'entitlements_ready';
+
+/**
+ * Modifies the input call back to chain parent notification of response.
+ * @param {!Window} win
+ * @param {!function()} callback
+ */
+export function createGoogleSigninCallback(win, callback) {
```

Comment:
Could you rename this function to "createGoogleSignInCallback"? I typically see the "i" from "in" capitalized, when people refer to Google Sign In (GSI)

Reply:
Done.